src/containers/shared/utils.js

```diff
@@ -78,6 +78,61 @@ const FORMAT_PRICE_DEFAULT_OPTIONS = {
   padding: 0,
 }
 
+export const isLaterVersion = (source, target) => {
```

Comment:
How does this method handle versions that include a commit hash?

Reply:
Is there a structural/comparison rule for that? I just followed the structure described here:
https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/BuildInfo.h#L46-L72

Reply:
As we discussed, versions with commit hash will be handled in a separate PRs to remove the hash.